### ISSUE TITLE
Allow for pre-defined environments

### ISSUE THREAD

**Issue by stuartshields** [bradp/generator-chassis-wp#2]
@bradp So you have support for pre-selected extensions to be installed, what I'm wondering is, could you have a list of predefined extensions for different environments? i.e: WordPress.com VIP would pre-select the VIP Classic extension, XDebug etc.

**Comment by bradp** [comment]
Good idea! I did something similar with [blueprints in vv](https://github.com/bradp/vv#blueprints), would make sense to have this for sure.

**Comment by stuartshields** [comment]
@bradp The other alternative I had was to fork this repo and create the VIP pre-selected extensions. However that would be overkill. 

IF we do go the path of creating this, be aware that we'll need to add https://github.com/stuartshields/vip-debugging to the list since currently the debugging extension causes issues because of mu-plugins in the VIP extension already having the rewrite rules extension.
